sysboundary/ionosphere.cpp

```diff
@@ -1320,14 +1346,21 @@ namespace SBC {
 
                  // Local cell
                  std::array<int,3> lfsc = technicalGrid.globalToLocal(fsc[0],fsc[1],fsc[2]);
-                 if(lfsc[0] == -1 && lfsc[1] == -1 && lfsc[2] == -1) {
+                 if(
+                    (lfsc[0] == -1 && lfsc[1] == -1 && lfsc[2] == -1)
+                   || (fsc[0] == 0 && fsc[1] == 0 && fsc[2] == 0)
```

Comment:
What do you think of this?

Reply:
I don't believe that is likely to ever trigger. The fsc value comes from fsgrid::globalToLocal, and if anything goes wrong, that would return -1, -1, -1:
https://github.com/fmihpc/fsgrid/blob/master/fsgrid.hpp#L398

Reply:
Ah, but actually, if there is any mixup and the fsgridCoupling is uninitialized, then this might be a problem. So in this case, that check would be a good idea.